ble/src/main/java/no/nordicsemi/android/ble/BleManagerCallbacks.java

```diff
@@ -74,8 +74,38 @@
 	 * Otherwise the {@link #onLinkLossOccurred(BluetoothDevice)} method will be called instead.
 	 *
 	 * @param device the device that got disconnected.
+	 * @deprecated Use {@link #onDeviceDisconnected(BluetoothDevice, int)} instead.
 	 */
-	void onDeviceDisconnected(@NonNull final BluetoothDevice device);
+	@Deprecated
+	default void onDeviceDisconnected(@NonNull final BluetoothDevice device) {
+		// do nothing
+	}
+
+	/**
+	 * Called when the device has disconnected (when the callback returned
+	 * {@link BluetoothGattCallback#onConnectionStateChange(BluetoothGatt, int status, int)} with
```

Comment:
Change(BluetoothGatt, int status, int)
should be (I think)
Change(BluetoothGatt, int, int)

Or did you add it for clarity?

Reply:
I did add it for clarity, but I'm closing the PR. There will be another one with more drastically changes.

Reply:
See https://github.com/NordicSemiconductor/Android-BLE-Library/pull/149#issuecomment-606750235.